documentation/src/main/wiki/Home.md

```diff
@@ -4,20 +4,15 @@
   <img src="https://github.com/Cognifide/knotx/blob/master/icons/180x180.png?raw=true" alt="Knot.x Logo"/>
 </p>
 
-A simplified description of **Knot.x** can be `a tool which converts a static page (template) into a 
-dynamic page driven by the data provided by external sources`
-
-In short, we call **Knot.x** a **reactive multisource assembler**.
+**Knot.x** is a high-efficient and scalable integration platform for modern websites written in [Vert.x](http://vertx.io/) technology.
 
 ## What problems does Knot.x solve?
-**Knot.x** assembles static and dynamic content from multiple sources to produce pages with dynamic data in a very performant manner.
 
-[[assets/knotx-high-level-architecture.png|alt=High Level Architecture]]
+**Knot.x** uses data from any source (like REST / SOAP service, search engine, CRM etc) and transforms it into an unified customer experience using a 
+template from a repository. The template can contain dynamic snippets which determine the way how data is used. The repository can be CMS system, Apache or 
+simple catalogue with static HTML pages. 
```

Comment:
"simple catalogue" is actually is to broad. Use something more understandable, e.g.
"any other system that's able to provide static HTML pages"